.github/workflows/pythonpublish.yml

```diff
@@ -61,27 +65,27 @@ jobs:
           mv nydus-static/* /usr/local/bin/
           rm -rf nydus-static-v2.2.4-linux-amd64.tgz nydus-static
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
+        uses: docker/setup-qemu-action@v3
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
+        uses: docker/setup-buildx-action@v3
 
       - name: Log in to the Container registry
-        uses: docker/login-action@v2
+        uses: docker/login-action@v3
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Extract metadata (tags, labels) for Docker
         id: meta
-        uses: docker/metadata-action@v4
+        uses: docker/metadata-action@v5
         with:
           images: |
-            ghcr.io/${{ github.repository }}
-            ghcr.io/owasp-dep-scan/depscan
+            ghcr.io/${{ github.repository }}:release-5.x
```

Comment:
Not sure if this is needed since the metadata action would compute some tags based on the branch and tags